bigquery/tables.rb

```diff
@@ -119,10 +114,9 @@ def import_table_data_from_file project_id:, dataset_id:, table_id:,
   # table_id        = "ID of the table to import file data into"
   # local_file_path = "Path to local file to import into BigQuery table"
 
-  require "google/cloud"
+  require "google/cloud/bigquery"
 
-  gcloud   = Google::Cloud.new project_id
-  bigquery = gcloud.bigquery
+  bigquery = Google::Cloud::Bigquery.new
```

Comment:
project: project_id is necessary because this is inside a function which has a project_id parameter.